utils/run_test.sh

```diff
@@ -67,7 +67,9 @@ JENKINS_TEST_LIST=(scripts/cart_echo_test.yml                   \
                    scripts/cart_test_no_pmix.yml                \
                    scripts/cart_test_no_pmix_multi_ctx.yml      \
                    scripts/cart_test_no_pmix_launcher.yml       \
-                   scripts/cart_test_no_timeout_non_sep.yml)
+                   scripts/cart_test_no_timeout_non_sep.yml     \
+                   scripts/cart_test_rpc_to_ghost_rank.yml      \
+                   scripts/cart_test_rpc_to_ghost_rank_non_sep.yml)
```

Comment:
isnt it just a single yaml file for both sep and non-sep? i dont see you adding _sep.yml file to this review